app/views/devise/sessions/new.html.erb

```diff
@@ -34,11 +34,14 @@
       ) %>
   <%= f.input :request_id, as: :hidden, input_html: { value: @request_id } %>
   <div class='margin-bottom-4'>
-    <%= f.submit t('links.next'), full_width: true, big: false, wide: false, class: 'margin-bottom-2' %>
+    <%= f.submit t('links.next'), full_width: true, big: false, wide: false, class: 'margin-bottom-3' %>
+    <h2 class='margin-top-1 margin-bottom-3 text-normal text-center text-center--line text-bold border-bottom'>
```

Comment:
Minor: I saw some commits which moved styles from the CSS to utility classes. I might actually be more in favor of the opposite, so that we can treat this more as a self-contained component.

e.g.

```html
<h2 class="separator-heading">
  <span class="separator-heading__text"><%= t('headings.create_account_with_sp.cta', app_name: APP_NAME) %></span>
</h2>
```